README.md

```diff
@@ -1,5 +1,13 @@
-Java Servlet API
-================
+# Jakarta Servlet
+
+This repository contains the code for Jakarta Servlet.
+
+[Online JavaDoc](https://javadoc.io/doc/jakarta.servlet/jakarta.servlet-api/)
+
+About Jakarta Servlet
+-------------
+￼
```

Comment:
Is there a control character at the end of this line?

Reply:
not that I could see, but line updated anyway/